internal/api/mfa.go

```diff
@@ -48,24 +52,79 @@ type EnrollFactorResponse struct {
 }
 
 type ChallengeFactorParams struct {
-	Channel string `json:"channel"`
+	Channel  string          `json:"channel"`
+	WebAuthn *WebAuthnParams `json:"web_authn,omitempty"`
 }
 
 type VerifyFactorParams struct {
-	ChallengeID uuid.UUID `json:"challenge_id"`
-	Code        string    `json:"code"`
+	ChallengeID uuid.UUID       `json:"challenge_id"`
+	Code        string          `json:"code"`
+	WebAuthn    *WebAuthnParams `json:"web_authn,omitempty"`
 }
 
 type ChallengeFactorResponse struct {
-	ID        uuid.UUID `json:"id"`
-	Type      string    `json:"type"`
-	ExpiresAt int64     `json:"expires_at"`
+	ID                        uuid.UUID                        `json:"id"`
+	Type                      string                           `json:"type"`
+	ExpiresAt                 int64                            `json:"expires_at"`
+	CredentialRequestOptions  *wbnprotocol.CredentialAssertion `json:"credential_request_options"`
+	CredentialCreationOptions *wbnprotocol.CredentialCreation  `json:"credential_creation_options"`
```

Comment:
```suggestion
	ExpiresAt                 int64                            `json:"expires_at,omitempty"`
	CredentialRequestOptions  *wbnprotocol.CredentialAssertion `json:"credential_request_options,omitempty"`
	CredentialCreationOptions *wbnprotocol.CredentialCreation  `json:"credential_creation_options,omitempty"`
```

I don't think ExpiresAt is used for WebAuthn?